microsetta_private_api/repo/metadata_repo/tests/test_repo.py

```diff
@@ -225,16 +263,17 @@ def test_to_pandas_dataframe(self):
         templates = {1: self.fake_survey_template2}
 
         exp = pd.DataFrame([['000004216', 'foo', UNSPECIFIED, 'No',
-                             'Unspecified', 'Unspecified', 'Unspecified', 'No',
+                             UNSPECIFIED, UNSPECIFIED, UNSPECIFIED, 'No',
                              'true', 'true', 'false', 'false',
                              UNSPECIFIED,
-                             'okay', 'No', "2013-10-15T09:30:00", '000004216'],
+                             'okay', 'No', "2013-10-15T09:30:00", '000004216',
+                             'US:CA', 'CA', '32', '-117'],
                             ['XY0004216', 'bar', 'Vegan foo', 'Yes',
-                             'Unspecified', 'Unspecified', 'Unspecified',
-                             'No', 'false', 'true', 'true', 'false', 'foobar',
-                             UNSPECIFIED,
-                             UNSPECIFIED,
-                             "2013-10-15T09:30:00", 'XY0004216']],
+                             UNSPECIFIED, UNSPECIFIED, UNSPECIFIED,
+                             'No', 'false', 'true', 'true', 'false',
+                             'foobar', UNSPECIFIED, UNSPECIFIED,
+                             "2013-10-15T09:30:00", 'XY0004216',
+                             'US:CA', 'CA', '32', '-117']],
```

Comment:
With the corrected round, this would be 33 

```suggestion
                             'US:CA', 'CA', '33', '-117']],
```